src/formatters/teamcity.ts

```diff
@@ -0,0 +1,53 @@
+import { Dictionary } from '@stoplight/types';
+import { IRuleResult } from '../types';
+import { Formatter } from './types';
+import { getSeverityName, groupBySource, sortResults } from './utils';
+
+function escapeString(str?: string | number) {
```

Comment:
```suggestion
function escapeString(str: Optional<string | number>) {
```
Let's make the argument required.